src/Verify/Etherscan.ts

```diff
@@ -140,7 +140,7 @@ export class Etherscan {
             )
         }
 
-        return await response.body.json()
+        return await response.body.json() as EtherscanResponseBody
```

Comment:
`response.body.json` was `unknow`, but that has nothing to do with the newer versions of the packages, I believe